### PR TITLE
feat(skyrim-platform): add API to contact back to game from browser

### LINKED ISSUE #235: Add API to contact back to the game from browser

### PR DESCRIPTION
**Pull request by Pospelove**
closes #235

### CODE REVIEW

File: skyrim-platform/src/third_party/cef/ui_process/ProcessHandler.cpp [hunk 1]
```diff
@@ -11,26 +11,10 @@ void ProcessHandler::OnContextCreated(CefRefPtr<CefBrowser> browser,
...
-  m_pCoreObject->SetValue(
-    "disconnect", CefV8Value::CreateFunction("disconnect", m_pOverlayHandler),
-    V8_PROPERTY_ATTRIBUTE_NONE);
   m_pCoreObject->SetValue(
     "sendMessage",
     CefV8Value::CreateFunction("sendMessage", m_pOverlayHandler),
     V8_PROPERTY_ATTRIBUTE_NONE);
-  m_pCoreObject->SetValue(
-    "deactivate", CefV8Value::CreateFunction("deactivate", m_pOverlayHandler),
-    V8_PROPERTY_ATTRIBUTE_NONE);
```
[the remark this answers is not in the document]
Comment (Pospelove) [reply skyrim-platform/src/third_party/cef/ui_process/ProcessHandler.cpp:33]: nothing, TiltedUI copy-paste

File: docs/docs_skyrim_platform.md [hunk 2]
```diff
@@ -353,6 +355,39 @@ browser.executeJavaScript("console.log('Hello CEF')");
...
+
+You can call this function whenever you want: as a button callback, etc.
+```html
+<input type="button" value="Click me" onclick="window.skyrimPlatform.sendMessage({ foo: 'bar' });">
+```
+
+Calls to `sendMessage` result in a `browserMessage` event on the SP side. You can handle these events as any others.
+```ts
+on("browserMessage", (event) => {
+  printConsole(JSON.stringify(event.arguments));
```
Comment (nic11) [inline docs/docs_skyrim_platform.md:375]: Is it game console (which opens with `)? BTW maybe add log level here to be future-proof? Probably optional, with INFO by default

File: docs/docs_skyrim_platform.md [hunk 3]
```diff
@@ -353,6 +355,39 @@ browser.executeJavaScript("console.log('Hello CEF')");
 const str = browser.getToken();
 ```
 
+#### Talking back to the game
+
+Use `window.skyrimPlatform.sendMessage` on the browser side to talk back to the game. `sendMessage` accepts zero or more JSON-serializable values.
```
Comment (nic11) [inline docs/docs_skyrim_platform.md:360]: IDK if it's good or bad practice in frontend development, but feels like explicitly writing `window.` is not needed

**Comment by Pospelove** [comment]
upd
circular

File: docs/docs_skyrim_platform.md (hunk 2 above)
Comment (Pospelove) [reply docs/docs_skyrim_platform.md:375]: Yes. Good idea. But it's out of PR's scope

File: docs/docs_skyrim_platform.md (hunk 3 above)
Comment (Pospelove) [reply docs/docs_skyrim_platform.md:360]: it matters, window object and global object are different

File: skyrim-platform/src/tilted/core_library/Filesystem.cpp [hunk 4]
```diff
@@ -0,0 +1,42 @@
+#include <Filesystem.hpp>
```
Comment (nic11) [inline skyrim-platform/src/tilted/core_library/Filesystem.cpp:1]: Wow. Now when I look at it, I see a lot of deleted and then added files unrelated to this PR. Maybe it would be better to make a separate PR with just renaming `third_party/cef` to `tilted`? History would be a lot cleaner

**Review by nic11 — APPROVED** [review]
